Simplenote/AuthViewController+Swift.swift

```diff
@@ -79,37 +74,55 @@ extension AuthViewController {
         switchActionButton.title   = switchText.uppercased()
     }
 
-    /// Displays / Hides components, based on the ViewController Mode (SignUp / LogIn)
+    /// Makes sure unused components (in the current mode) are effectively disabled
     ///
-    func refreshVisibleComponentsWithAnimation() {
-        let mustHide = !signingIn
+    func refreshEnabledComponents(signingIn: Bool) {
```

Comment:
`signingIn` is a property on `AuthViewController`, do we really need to pass it around? Will it happen that we will pass a value here that won't match `self.signingIn`? 🤔

Reply:
Too much Erlang. You're right, it makes no sense to actually pass it around!